client/src/App.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import './App.css';
+import MainSidebar from './components/MainSidebar';
 
 export default () => (
   <BrowserRouter>
     <Switch>
       <Route exact path="/" component={() => <div />} />
+      <Route exact path="/main-sidebar" component={() => <MainSidebar selected="tickets" />} />
```

Comment:
I've never seen the component used like that (calling an arrow function that returns the component). I can't find the difference but it looks cleaner to just use the component name directly.